platform/default/include/mbgl/storage/offline_schema.sql

```diff
@@ -1,55 +1,156 @@
-CREATE TABLE resources (                   -- Generic table for style, source, sprite, and glyph resources.
-  id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-  url TEXT NOT NULL,
-  kind INTEGER NOT NULL,
-  expires INTEGER,
-  modified INTEGER,
-  etag TEXT,
-  data BLOB,
-  compressed INTEGER NOT NULL DEFAULT 0,
-  accessed INTEGER NOT NULL,
-  must_revalidate INTEGER NOT NULL DEFAULT 0,
+--
+-- Table containing the style, source, sprite, and glyph
+-- resources. Essentially everything that is not a tile.
+--
+CREATE TABLE resources (
+  id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,   -- Primary key.
+
+  url TEXT NOT NULL,                               -- The URL of the resource without the access token. If a Mapbox
+                                                   -- resource, will be stored using the mapbox:// schema. Must be
+                                                   -- unique and that is enforced by the database schema.
+
+  kind INTEGER NOT NULL,                           -- Type of the resource, taken from Resource::Kind enumeration:
+                                                   -- style        = 1
+                                                   -- source       = 2
+                                                   -- tile         = 3
+                                                   -- glyphs       = 4
+                                                   -- sprite image = 5
+                                                   -- sprite JSON  = 6
+                                                   -- image        = 7
+
+  expires INTEGER,                                 -- Expiration time. The resource will be refreshed after this
+                                                   -- expiration is reached.
+
+  modified INTEGER,                                -- Last time the resource was modified.
+
+  etag TEXT,                                       -- Checksum used for cache optimization. If, when refreshing the
+                                                   -- resource, it matches the etag on the server, the resource will
+                                                   -- not get re-downloaded. See:
+                                                   -- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag
+
+  data BLOB,                                       -- Contents of the resource.
```

Comment:
Just double checking, should the documentation on this line and line 75 be different?

Reply:
If so, why? Most other things are the same